atest/robot/libdoc/doc_format.robot

```diff
@@ -67,6 +86,9 @@ Test Format In XML
     Format should be    ${expected}
     Keyword Doc Should Be    0    *bold* or <b>bold</b> http://example.com
 
-Format should be
-    [Arguments]    ${expected}
-    Element Attribute Should Be    ${LIBDOC}    format    ${expected}
+Test Format In XML:HTML
+    [Arguments]    ${expected}    ${cli}=    ${lib}=DocFormat.py
+    ${lib} =    Join Path    ${TESTDATADIR}    ${lib}
+    Run Libdoc And Parse Output     ${cli} ${lib}
+    Format should be    ${expected}
```

Comment:
Isn't this format always HTML? If yes, it should be hard coded here and the `${expected}` argument removed.

Reply:
thats correct.
i added the `${expected}` to the arguments to have a better self similarity.
That a person who tries to understand the test not always habe to look into Keyword definition.

If you insist, i can remove it.

Reply:
An argument that isn't really used feels, well, useless for me. It also gives an impression that XML:HMTL format would support some other doc formats than HTML. I'd prefer this to be changed but don't feel strongly. If you prefer this format, leave it like that.

Reply:
An argument that isn't really used feels, well, useless for me. It also gives an impression that XML:HMTL format would support some other doc formats than HTML. I'd prefer this to be changed but don't feel strongly. If you prefer this format, leave it like that.